CODEOWNERS

```diff
@@ -1 +1 @@
-* @brikis98  @pete0emerson
```

Comment:
imho remove all, way to much formalism for such repo :)

Reply:
Um... this is such a critical repo for so many modules.  Why do you think it needs LESS safeguards?

Reply:
this codeowners file is repo wide, not per folder. 
any repowide codeowners can be done as gh repo config. right?
codeowners is useless here.

also codeowners does not state who allowed to modify code,
but gh tf config does.
more people can modify than listed in codeowners.

Reply:
see https://github.com/gruntwork-io/bash-commons/pull/64

in reality none codeowners looked or approved and yet merged.
so codeowners useless process here.

codeowners mostly for monorepos.

Reply:
not working sadeguard is LESS safeguards actually. 
removing will add more by not obscuring reality.

so i suggest more security actually;) also gh tf.